variables.tf

```diff
@@ -15,6 +15,12 @@ variable "location" {
   default     = "us-east-1"
 }
 
+variable "azs" {
+  description = "A list of availability zones names or ids in the region"
+  type = list(string)
+  default = null
+}
+
```

Comment:
This does not belong here. The `azs` flag should be in the [`../modules/aws_vpc`](https://github.com/sassoftware/viya4-iac-aws/blob/main/modules/aws_vpc/variables.tf) variables.tf file

Reply:
I believe this variable must be surfaced at root level, for the user to enter list of desired AZs.